lib/rules/jsx-props-no-spreading.js

```diff
@@ -31,44 +31,49 @@ module.exports = {
       recommended: false,
       url: docsUrl('jsx-props-no-spreading')
     },
-    schema: [{
-      allOf: [{
-        type: 'object',
-        properties: {
-          html: {
-            enum: [OPTIONS.enforce, OPTIONS.ignore]
-          },
-          custom: {
-            enum: [OPTIONS.enforce, OPTIONS.ignore]
-          },
-          exceptions: {
-            type: 'array',
-            items: {
-              type: 'string',
-              uniqueItems: true
+    schema: [
+      {
```

Comment:
can you please revert the unrelated formatting changes here?

Reply:
Done